core/amber/src/main/protobuf/edu/uci/ics/amber/engine/architecture/rpc/controlcommands.proto

```diff
@@ -272,4 +273,8 @@ message PrepareCheckpointRequest{
 
 message QueryStatisticsRequest{
   repeated core.ActorVirtualIdentity filterByWorkers = 1;
+}
+
+message EndWorkerRequest {
+  string actorRefStr = 1;
```

Comment:
I don't find the usage of this actorRefStr. is it needed? If not please remove it.